src/internal/connector/onedrive/collections.go

```diff
@@ -608,6 +608,13 @@ func (c *Collections) UpdateCollections(
 			break
 		}
 
+		if item.GetMalware() != nil {
+			// TODO: track the item as skipped; logging alone might
+			// slice out the data from tracking.
+			logger.Ctx(ctx).Infow("malware detected", "malware_description", item.GetMalware().GetDescription())
```

Comment:
From the [docs](https://learn.microsoft.com/en-us/onedrive/developer/rest-api/resources/malware?view=odsp-graph-online), malware does not have any additional fields within it, unless this is a field filled in by the sdk. I did not try desc for malware but other fields with this method seemed to return nil.

Reply:
https://learn.microsoft.com/en-us/graph/api/resources/malware?view=graph-rest-1.0